src/output/RepresenterRewriteOutput.ts

```diff
@@ -18,6 +19,7 @@ import {
 
 type Program = TSESTree.Program
 type Node = TSESTree.Node
+type LiteralNode = { type: string, value: string|boolean|null|number, raw?: string }
```

Comment:
Could `type` also be a union type?

Reply:
I left is as a string because I think that's what it is in the interface for Node that's coming from astring.
I guess it can become a `string|null`?

Reply:
What I was sort of trying to get at is that I would expect `type` to be a fixed list of types, as there must be a fixed number of different literal node types. But I take it we don't have that information?

Reply:
As far as I understood, there's only `Literal` and `TemplateLiteral` nodes that are used for literals. Just the `Literal` node's value is of different type, either string, number, boolean etc. 
I could probably do `type: Literal|TemplateLiteral` but that excludes all other types, and to be honest I don't think it's feasible to list them all

Reply:
Okay

Reply:
so, leaving it as is now?

Reply:
Okay